### ISSUE TITLE
Login function error in gallery's twitter example

### ISSUE THREAD

**Issue by dongshik** [pynecone-io/pynecone#531]
**Describe the bug**
In the login screen of the gallery's twitter example, the event handler does not work when attempting to log in again after failing to log in once.
All events seem to stop after return pc.window_alert("Invalid username or password.") in the login state function.
So I removed the alert part and treated it as a message.

```
 class AuthState(State):
    password: str
    confirm_password: str
    message: str

...

    def login(self):
        """Log in a user."""
        with pc.session() as session:
            user = session.exec(
                User.select.where(User.username == self.username)
            ).first()
            if user and user.password == self.password:
                self.logged_in = True
                return pc.redirect("/home")
            else:
                self.message="Invalid username or password."
                #return pc.window_alert("Invalid username or password.")
                return
...

def login():
    return pc.box(
        pc.vstack(
            navbar(State),
            pc.center(
                pc.vstack(
                    pc.input(
                        on_blur=State.set_username, placeholder="Username", width="100%"
                    ),
                    pc.input(
                        on_blur=AuthState.set_password,
                        placeholder="Password",
                        type_="password",
                        width="100%",
                    ),
                    pc.button("Login", on_click=AuthState.login, width="100%"),
                    pc.link(
                        pc.button("Sign Up", width="100%"), href="/signup", width="100%"
                    ),
                    pc.text(AuthState.message, color="red"),
                ),
                style=styles["login_input"],
            ),
        ),
        style=styles["login_page"],
    )

```
![image](https://user-images.githubusercontent.com/7720485/218442339-f9548de8-5cc9-42fe-abee-a8f1a8baa430.png)

**To Reproduce**

the first login fails with the wrong password
second retry login again 

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/7720485/218439161-77417c34-822a-438a-902f-b473494420e0.png)

** Specifics (please complete the following information):**
 - Python Version: Python 3.9.6
 - Pynecone Version: 0.1.15
 - OS: iOS
 - Browser (Optional): 
 
**Additional context**

**Comment by karndeb** [comment]
Known issue. https://github.com/pynecone-io/pynecone/issues/411
Fix https://github.com/pynecone-io/pynecone/pull/465
Will be fixed in the v0.1.16

**Comment by dongshik** [comment]
>
